eng/pipeline/cleanup-acr-images.yml

```diff
@@ -0,0 +1,75 @@
+# This pipeline is used to clean up old images in our Azure Container Registries.
+# It's derived from https://github.com/dotnet/docker-tools/blob/main/eng/pipelines/cleanup-acr-images.yml
+
+trigger: none
+pr: none
+
+schedules:
+- cron: "0 5 * * *"
+  displayName: Nightly build
+  branches:
+    include:
+    - microsoft/main
+  always: true
+
+variables:
+- template: /eng/pipelines/templates/variables/common.yml@self
+
+extends:
+  template: /eng/common/templates/1es-official.yml@self
+  parameters:
+    serviceConnections:
+    - name: $(clean.serviceConnectionName)
+    stages:
+    - stage: Build
+      dependsOn: []
+      jobs:
+      - job: Build
+        steps:
+        - template: /eng/common/templates/steps/init-docker-linux.yml@self
+        - template: /eng/common/templates/steps/clean-acr-images.yml@self
+          parameters:
+            internalProjectName: ${{ variables.internalProjectName }}
+            repo: "build-staging/*"
+            subscription: $(acr-staging.subscription)
+            resourceGroup: $(acr-staging.resourceGroup)
+            acr: $(acr-staging.server)
+            action: delete
+            age: 15
+        # - template: /eng/common/templates/steps/clean-acr-images.yml@self
+        #   parameters:
+        #     internalProjectName: ${{ variables.internalProjectName }}
+        #     repo: "public/dotnet/*"
```

Comment:
we may want to do something similar here? I'm guessing we'd set repo as `public/oss/go/*`

Reply:
I don't think that's a great idea until we figure out what's going on with:

* https://github.com/microsoft/go-lab/issues/245

Concerned that we're actually hitting undefined behavior when it comes to EOL detection, perhaps good enough for marking but deleting is probably riskier.